<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FournisseurRechercherWidget</class>
 <widget class="QWidget" name="FournisseurRechercherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rechercher</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Code Fournisseur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCle"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRechercher">
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>1</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>Num Fournisseuri</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom Fourn</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Num Fournisseur</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Registre de commerce</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telephone</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fax</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>C Corespondance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Site Web</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fichier</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Adresse Banque</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Compte B</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Adresse Fournisseur</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Adresse</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Adresse 1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Remarque 1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Remarque 2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Flag</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonModifier">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Modifer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSupprimer">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
